### PR TITLE
refactor: Changes package name to @digital-uk prefix

### PR DESCRIPTION
**Pull request by rathermarv**
Updates `package.json` and `package-lock.json` to reflect move to Digital UK namespace.

This is the first in a chain of PRs that refactor and expand upon the source repo to support existing apps, ATVG and xFVP.
However the intention is that this will be used in fvp-utils and possibly re-exported from there. As such, it will underpin applications that are developed against a growing framework. For that reason I've added a few of reviewers to, if nothing else, flag up the work that's coming and provide some foresight.

### CODE REVIEW

File: package.json
Comment (joelKnapp) [inline package.json:14]: This makes it look like philippsimon is a DUK employee

Comment (rathermarv) [reply package.json:14]: Hmm, it does rather, I don't know if there's a convention for forked npm projects..?

Comment (joelKnapp) [reply package.json:14]: changing the author to Digital UK and adding a contributors field with philippsimon?

Comment (joelKnapp) [reply package.json:14]: it will also always say in GitHub that it was forked and give a link to the original

Comment (rathermarv) [reply package.json:14]: Looks good - will make the change.
@joelKnapp by convention would you or I resolve the convesation when this change is made?

Comment (joelKnapp) [reply package.json:14]: I'll resolve, we should turn on the require conversations to be resolved option in the project, and consider turning on the auto dismiss stale reviews

Comment (rathermarv) [reply package.json:14]: All sounds good. I think we need to hang on until David's around again to clarify that a public fork of this repo is what's actually wanted, but then definitiely would be good to get the repo settings all twiddled with.

**Review by joelKnapp — APPROVED** [review]

File: package.json
Comment (parkeyparker) [inline package.json:17]: Would this not be the other way round? Digital UK are contributing to something originally created by philippsimon...

Comment (rathermarv) [reply package.json:17]: > Would this not be the other way round? Digital UK are contributing to something originally created by philippsimon...

I got my steer [here](https://opensource.stackexchange.com/questions/8242/authors-and-contributors-of-forked-project).
I think this probably makes sense this way around. From hereon in, Digital UK is fundamentally in control of what's in and what's out.

Comment (rathermarv) [reply package.json:17]: Am hanging on before merging though, in case there's any more debate to be had....

Comment (parkeyparker) [reply package.json:17]: The part I hadn't considered is that most people would look to the author to contact for issues and so the author should indeed be Digital-UK so I think this is good as is now.

**Review by parkeyparker — APPROVED** [review]

**Pull request merged**
